projects/lldpd/build.sh

```diff
@@ -14,7 +14,11 @@
 # limitations under the License.
 #
 ################################################################################
+
 ./autogen.sh
+
+export check_LIBS="/usr/lib/x86_64-linux-gnu/libcheck.a /usr/lib/x86_64-linux-gnu/libsubunit.a -lrt -lm -lpthread"
```

Comment:
can we avoid the absolute path? e.g. "-l:libcheck.a -l:libsubunit.a"?

Reply:
Fixed.